src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrScanner.cs

```diff
@@ -79,12 +82,59 @@ private async Task<string[]> ScanRepository(SourceMapping sourceMapping, Cancell
 
         foreach (var file in files)
         {
-            _logger.LogWarning("File {file} is cloaked but present in the VMR", file);
+            _logger.LogWarning("File in {file} is cloaked but present in the VMR", file);
```

Comment:
I am thinking about how we will use this.

Would it make sense that with the default logging verbosity, we only get a list of files out? So that we can parse those later in a pipeline or wherever?
And if people call `--verbose` or `--debug` they would get more logging around.

I can imagine we might end up exporting the list of offending files as a pipeline artifact for some baseline or so..

Reply:
The default for the logger is set that it only outputs errors, if the `verbose` or `debug` flags are not set. I think this wouldn't play nice with AzDo.
I guess we could overwrite the logger for these `scan` operations

Reply:
It outputs warnings+errors, no?

Reply:
https://github.com/dotnet/arcade-services/blob/6f954ccfbe6a51861836a1f92184d7a6c6060f43/src/Microsoft.DotNet.Darc/src/Darc/Operations/Operation.cs#L27-L35
https://github.com/dotnet/arcade-services/blob/6f954ccfbe6a51861836a1f92184d7a6c6060f43/src/Microsoft.DotNet.Darc/src/Darc/Operations/Operation.cs#L43-L46

Ahh  yes, my bad

Reply:
Done